From ca0d524c44d9b2e575dd52a2b13c42f1f44da059 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Wed, 11 Apr 2018 17:03:28 -0500
Subject: [PATCH] Use "CoreSetupRid" to redist

Make sure Microsoft.NETCore.DotNetAppHost, for example, can be restored.
---
 build/AppHostTemplate.proj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/AppHostTemplate.proj b/build/AppHostTemplate.proj
index 598f250b6..ba80aa405 100644
--- a/build/AppHostTemplate.proj
+++ b/build/AppHostTemplate.proj
@@ -35,7 +35,7 @@
 
   <Target Name="EnsureAppHostPackageRestored">
     <PropertyGroup>
-      <AppHostTemplateRestoreAdditionalParameters>--runtime $(Rid)</AppHostTemplateRestoreAdditionalParameters>
+      <AppHostTemplateRestoreAdditionalParameters>--runtime $(CoreSetupRid)</AppHostTemplateRestoreAdditionalParameters>
       <AppHostTemplateRestoreAdditionalParameters>$(AppHostTemplateRestoreAdditionalParameters) /p:TargetFramework=$(CliTargetFramework)</AppHostTemplateRestoreAdditionalParameters>
       <AppHostTemplateRestoreAdditionalParameters>$(AppHostTemplateRestoreAdditionalParameters) /p:TemplateFillInPackageName=$(TemplateFillInPackageName)</AppHostTemplateRestoreAdditionalParameters>
       <AppHostTemplateRestoreAdditionalParameters>$(AppHostTemplateRestoreAdditionalParameters) /p:TemplateFillInPackageVersion=$(TemplateFillInPackageVersion)</AppHostTemplateRestoreAdditionalParameters>
-- 
2.16.1.windows.4

